app/components/operator-form/template.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="input-form-wrapper">
 {{#unless singleOperator}}
-	<div class="row">
+	<div class="{{if operator.include_in_changeset "yes"}} {{unless operator.include_in_changeset "no"}} row">
```

Comment:
I tried using {{#if}} {{else}}{{/if}} , but it doesn't pick up else condition, maybe I wrote wrong syntax? it would be better to change this to if/else .